hibernate/cache-commons/pom.xml

```diff
@@ -45,7 +45,7 @@
       <dependency>
          <groupId>org.jboss.narayana.jta</groupId>
          <artifactId>narayana-jta-jakarta</artifactId>
-         <version>5.11.2.Final</version>
+         <version>${version.jboss.narayana.javaee}</version>
```

Comment:
what is the error using 6.0.0.Final?

Reply:
No idea. It was already set to `5.11.2.Final` before this PR. I have reverted back to this.

Reply:
It looks like it is broken in `main`

```
[INFO] org.infinispan:infinispan-hibernate-cache-commons:jar:15.0.0-SNAPSHOT
[INFO] +- org.jboss.narayana.jta:narayana-jta-jakarta:jar:5.11.2.Final:test
[INFO] \- org.jboss.narayana.jta:narayana-jta:jar:5.13.1.Final:test
```
```
[INFO] --- dependency:3.1.1:tree (default-cli) @ infinispan-hibernate-cache-spi ---
[INFO] org.infinispan:infinispan-hibernate-cache-spi:jar:15.0.0-SNAPSHOT
[INFO] \- org.jboss.narayana.jta:narayana-jta:jar:5.13.1.Final:test
```
```
[INFO] org.infinispan:infinispan-hibernate-cache-v62:jar:15.0.0-SNAPSHOT
[INFO] +- org.jboss.narayana.jta:narayana-jta-jakarta:jar:5.13.1.Final:test
[INFO] \- org.jboss.narayana.jta:narayana-jta:jar:6.0.0.CR1:test
```

Reply:
Thanks @pruivo, I have applied your suggestion but removed `narayana-jta-jakarta` entirely as that does not exist for 6.0.x

Reply:
Right, Narayana dropped javaee! :+1: